### PR TITLE
Org.gnu.emacs

### PR DESCRIPTION
**Pull request by muep**
Discussion with upstream on intent to make this flatpak available:
http://lists.gnu.org/archive/html/emacs-devel/2018-04/msg00507.html

In short, so far one on the mailing list seemed to immediately want to take this work. I am planning to collaborate on this with @zoglesby.

Please let me know if the initial pull request should provide a cleaner history than this current, rather honest one.

### CODE REVIEW

**Comment by muep** [comment]
bot, build

File: org.gnu.emacs.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "emacs",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=pulseaudio",
+        "--socket=x11",
+        "--filesystem=host",
+        "--filesystem=/tmp"
```
Comment (barthalion) [inline org.gnu.emacs.json]: What's wrong with using private TMPDIR?

File: org.gnu.emacs.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                    "path": "appdata-releases.patch"
+                }
+            ],
+            "cleanup": [
+                "/share/icons/hicolor/scalable/mimetypes/emacs-document.svg",
+                "/share/icons/hicolor/scalable/mimetypes/emacs-document23.svg"
+            ]
+        }
+    ],
+    "rename-icon": "emacs",
```
Comment (barthalion) [inline org.gnu.emacs.json]: Really a nitpick, but it makes more sense to have these on top if the file.

**Comment by barthalion** [comment]
bot, build org.gnu.emacs

**Comment by nedrichards** [comment]
bot, build org.gnu.emacs

File: org.gnu.emacs.json (hunk 1 above)
Comment (muep) [reply org.gnu.emacs.json]: I just noticed that sudo -e places the temporary copy of the file into /var/tmp so we might want also that.

**Comment by barthalion** [comment]
It has built successfully: https://flathub.org/builds/#/builders/2/builds/1519

Looks good to me. Ready to merge?

**Comment by muep** [comment]
I am not sure if the question was for me, but it is ready as far as I am concerned. Thank you for reviewing.

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/org.gnu.emacs

Thanks!

**Pull request closed without merging**